JavaTutorialPrograms/java/sortNumber/SortNumber.java

```diff
@@ -0,0 +1,36 @@
+package sortNumber;
+
+public class SortNumber {
+    //Write program for sort number
+    public static void main(String[] args) {
+        int[] array = new int[]{1, 5, 7, 9, 3, 2};
+        length = array.length;
+        System.out.print("Input is:");
+        printArray(array);
+        sortArray(array);
+    }
+
+    static int length;
+
+    public static void printArray(int[] array) {
+        for (int counter = 0; counter < length; counter++) {
+            System.out.print(array[counter] + " ");
+        }
```

Comment:
No need to use for loop to print Array values, you can use Arrays.toString() method